db/schema.rb

```diff
@@ -54,6 +96,143 @@
+  end
+
+  create_table "people", force: :cascade do |t|
+    t.string "first_name"
+    t.string "last_name"
+    t.string "phone"
+    t.string "email"
+    t.string "phone_2"
+    t.string "email_2"
+    t.string "preferred_contact_method"
+    t.string "preferred_contact_timeframe"
+    t.text "skills"
+    t.text "notes"
+    t.bigint "user_id", null: false
+    t.datetime "created_at", precision: 6, null: false
+    t.datetime "updated_at", precision: 6, null: false
+    t.index ["user_id"], name: "index_people_on_user_id"
+  end
+
+  create_table "positions", force: :cascade do |t|
+    t.bigint "location_id", null: false
+    t.bigint "person_id", null: false
+    t.string "position_type"
+    t.string "name"
+    t.string "description"
+    t.date "start_date"
+    t.date "end_date"
+    t.boolean "is_primary"
+    t.text "notes"
+    t.datetime "created_at", precision: 6, null: false
+    t.datetime "updated_at", precision: 6, null: false
+    t.index ["location_id"], name: "index_positions_on_location_id"
+    t.index ["person_id"], name: "index_positions_on_person_id"
+  end
+
+  create_table "shared_accounts", force: :cascade do |t|
+    t.string "name"
+    t.string "account_url"
+    t.string "username"
+    t.string "password_hint"
+    t.string "notes"
+    t.datetime "created_at", precision: 6, null: false
+    t.datetime "updated_at", precision: 6, null: false
+  end
+
+  create_table "system_locations", force: :cascade do |t|
+    t.bigint "parent_id", null: false
+    t.bigint "organization_id", null: false
+    t.string "location_type"
+    t.string "street_address"
+    t.string "city"
+    t.string "state"
+    t.string "zip"
+    t.string "county"
+    t.string "region"
+    t.string "neighborhood"
+    t.datetime "created_at", precision: 6, null: false
+    t.datetime "updated_at", precision: 6, null: false
+    t.index ["organization_id"], name: "index_system_locations_on_organization_id"
+    t.index ["parent_id"], name: "index_system_locations_on_parent_id"
+  end
+
+  create_table "system_settings", force: :cascade do |t|
+    t.string "exchange_type"
+    t.boolean "separate_asks_offers"
+    t.boolean "allow_sms"
+    t.boolean "community_resources_module"
+    t.boolean "announcements_module"
+    t.boolean "positions_module"
+    t.boolean "donations_module"
+    t.boolean "shared_accounts_module"
+    t.boolean "chat_module"
+    t.text "landing_page_text_what"
+    t.text "landing_page_text_who"
+    t.text "landing_page_text_how"
+    t.datetime "created_at", precision: 6, null: false
+    t.datetime "updated_at", precision: 6, null: false
+  end
+
+  create_table "system_tags", force: :cascade do |t|
+    t.bigint "organization_id"
+    t.bigint "parent_id"
+    t.string "parent_type"
+    t.string "name", null: false
+    t.string "description"
+    t.boolean "display_to_public", default: true, null: false
+    t.integer "display_order", default: 10, null: false
+    t.string "created_by"
```

Comment:
Is this for auditing?

Reply:
Yes. I think we should consider a gem or system-wide auditing option, but this was a start to prompt that topic.